unit/src/main/cpp/subsystems/display.cpp

```diff
@@ -165,6 +171,33 @@ void Display::calculate(State *state, Commands *commands, Outputs *outputs) {
         outputs->logVal = state->camTvec[0];
     }
 
+    // Camera move
+    double sigmoidDD =
+        100 * -(std::cos(3.1415926535 * (state->indexCameraMove / 100.0)) - 1) /
+        2;
+
+    //    std::cout << outputs->positionCamera[0] << std::endl;
+    //    std::cout << outputs->targetCameraPosition[0] << std::endl;
+    //    std::cout << outputs->previousCameraPosition[0] << std::endl;
+
+    outputs->positionCamera[0] = (((outputs->targetCameraPosition[0] -
+                                    outputs->previousCameraPosition[0]) /
+                                   100.0) *
+                                  sigmoidDD) +
+                                 outputs->previousCameraPosition[0];
+    outputs->positionCamera[1] = (((outputs->targetCameraPosition[1] -
+                                    outputs->previousCameraPosition[1]) /
+                                   100.0) *
+                                  sigmoidDD) +
+                                 outputs->previousCameraPosition[1];
+    outputs->positionCamera[2] = (((outputs->targetCameraPosition[2] -
+                                    outputs->previousCameraPosition[2]) /
+                                   100.0) *
+                                  sigmoidDD) +
+                                 outputs->previousCameraPosition[2];
+
```

Comment:
could possibly clean this up using a for loop? it's probably find if you don't do this